src/collections/Form/index.d.ts

```diff
@@ -173,7 +173,7 @@ interface FormGroupProps {
 
 export const FormGroup: React.ComponentClass<FormGroupProps>;
 
-interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
+interface FormInputProps extends InputBaseProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
```

Comment:
Hm, why exclude the `onChange` prop from the FormInputProps?  All Input props are supported in the Form control shorthand components.  I may be missing something here.